curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e55b4b06c11fff555c64.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-77
 
 # --description--
 
-You need to know if the user's calories are a surplus or a deficit. Declare a `surplusOrDeficit` variable.
+You need to know if the user is in a caloric surplus or deficit. A caloric surplus is when you consume more calories than you burn, and a caloric deficit is when you burn more calories than you consume. Burning as many calories as you consume is called maintenance, and can be thought of as a surplus or deficit of `0`, depending on your goals.
```

Comment:
description was updated based on the suggestion notes and messages from the original issue